packages/adena-extension/src/containers/transfer-summary-container/transfer-summary-container.tsx

```diff
@@ -25,12 +26,19 @@ const TransferSummaryContainer: React.FC = () => {
   const { currentAccount, currentAddress } = useCurrentAccount();
   const navigate = useNavigate();
   const [summaryInfo, setSummaryInfo] = useState<TransferSummaryInfo>(state)
-  const [sendable, setSendable] = useState(true);
+  const [isSend, setIsSend] = useState(false);
```

Comment:
If the variable is of type bool, indicating whether it was sent or not, `isSent` is a more appropriate name.

Reply:
The variable names you suggested seem to be correct.